templates/sitemap.xml.twig

```diff
@@ -17,6 +19,22 @@
 {% endif %}
 {% if entry.priority %}
     <priority>{{ entry.priority|number_format(1) }}</priority>
+{% endif %}
+{% if entry.images %}
+{% for image in entry.images %}
+    <image:image>
+      <image:loc>{{ image.loc }}</image:loc>
+{% if image.caption %}
+      <image:caption>{{ image.caption }}</image:caption>
+{% endif %}
+{% if image.geoloc %}
+      <image:geo_location>{{ image.geoloc }}</image:geo_location>
+{% endif %}
+{% if image.title %}
+      <image:title>{{ image.title }}</image:title>
+{% endif %}
+    </image:image>
+{% endfor %}
 {% endif %}
```

Comment:
Pretty sure you don't need the surrounding `if` here, the loop won't run or throw an error if `entry.images` isn't defined.